client/src/modules/lead/EditLead/EditLeadContent/EditLeadTabs/EditLeadTabs.tsx

```diff
@@ -0,0 +1,163 @@
+    fakeInputContent: ' take notes',
+    showFakeInput: true,
+  };
+
+  public componentDidMount() {
+    if (this.state.activeTab === null) {
+      this.setState({ activeTab: <EditLeadEditor height={160} onCancel={this.onCancel} onSave={this.onNoteSave} /> });
+    }
+  }
+
+  public tabHandler = (content: any) => {
+    this.setState({
+      activeTab: content, showFakeInput: false,
+    });
+    this.fakeHandler(content);
+  }
+
+  public fakeHandler = (activeTab: any) => {
+
+    switch (activeTab.type) {
+      case (this.getNoteEditor().type): {
+        this.setState({ fakeInputContent: ' take notes' });
+        break;
+      }
+      case (this.getAddActivity().type): {
+        this.setState({ fakeInputContent: ' add activity' });
+        break;
+      }
+      default: {
+        this.setState({ fakeInputContent: ' take notes' });
+        break;
+      }
+
+    }
+  }
+
+  public onNoteSave = (noteText: string) => {
+    const note = {
+      text: noteText,
+      user: this.props.userId,
+    };
+    this.props.createNote(this.props.editLead._id, note);
+    this.setState({ showFakeInput: true });
+  }
+
+  public onActivitySave = (activity: Activity) => {
+    this.props.createActivity({
+      ...activity,
+      assignedTo: this.props.userId,
+      lead: this.props.editLead._id,
+    });
+    this.setState({ showFakeInput: true });
+  }
+
+  public onCancel = () => {
+    this.setState({ showFakeInput: true });
+  }
+
+  public render() {
+    const takeNotesCondition = this.isActive(EditLeadEditor, this.state.activeTab) || isBlank(this.state.activeTab);
+    const addActivityCondition = this.isActive(AddActivity, this.state.activeTab);
+
+    return (
+      <div className={styles.tabs}>
+        <ul className={styles.header}>
+          <li
+            className={classNames(styles.headerItem, { [styles.active]: takeNotesCondition })}
+            onClick={this.tabHandler.bind(this, this.getNoteEditor())}>
+            <img
+              src={takeNotesCondition ? takeNotesIconActive : takeNotesIcon}
+              className={styles.headerItemIcon}
+              alt="take note icon"
+            />
+            Take notes
+          </li>
+          <li
+            className={classNames(styles.headerItem, { [styles.active]: addActivityCondition })}
+            onClick={this.tabHandler.bind(this, this.getAddActivity())}>
+            <img
+              src={addActivityCondition ? addActivityIconActive : addActivityIcon}
+              className={styles.headerItemIcon}
+              alt="add activity icon"
+            />
+            Add activity
+          </li>
+        </ul>
+        {this.renderFakeInput(this.state.showFakeInput, takeNotesCondition)}
+      </div>
+    );
+  }
+  private isActive = (component: any, activeTab: any) => {
+    return (activeTab ? activeTab.type : null) === component;
+  }
+
+  private renderFakeInput = (condition: boolean, notesCondition: boolean) => {
+    if (condition) {
```

Comment:
you can get this condition directly from `this.state`, considering that this function is already arrow